eventing/samples/cronjob-source/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron Job Source example
+
+Cron Job Source example shows how to configure Cron Job as event source for functions.
+
+## Deployment Steps
+
+### Prerequisites
+
+1. Setup [Knative Serving](https://github.com/knative/docs/tree/master/serving).
+1. Setup
+   [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+
+### Channel
+
+1. Create a `Channel`. You can use your own `Channel` or use the provided sample, which creates a channel called `cronjob-test`. If you use your own `Channel` with a different name, then you will need to alter other commands later.
+
+```shell
+kubectl apply -f channel.yaml
+```
+
+### Create Cron Job Event Source
+
+1. In order to receive events, you need to create a concrete Event Source for a specific namespace. If you want to consume events from a differenet namespace or using a different `Service Account`, you need to modify the yaml accordingly.
```

Comment:
```suggestion
1. For each set of cron events you want to request, you need to create a Event Source in a the same namespace as the destiantion. If you need a different ServiceAccount to create the Deployment, modify the entry in `cronjob-source.yaml`.
```

Reply:
I think the tech writing team has also been encouraging actually including the yaml content inline for short and medium samples. The content reads better that way, instead of forcing users to bounce between different pages.